ActindoWorkFlow/Operation/01_ManageWorkflows.md

```diff
@@ -1,29 +1,33 @@
 [!!Tasks](../../Tasks/Overview/01_General.md)
 [!!User interface Workflows](../UserInterface/02a_Workflows.md)
 [!!Workflow and process elements](../Overview/04_WorkflowProcessElements.md)
-[!!Track a workflow process](./02_TrackWorkflowProcess.md)
+[!!Compare the workflow versions](./03_CompareWorkflows.md)
+[!!Copy a workflow](./04_CopyWorkflow.md)
+[!!Update a deployed version](05_UpdateDeployedVersion.md)
+[!!Insert a static input](06_InsertStaticInput.md)
+[!!Track a workflow process](./09_TrackWorkflowProcess.md)
 
 # Manage the workflows
 
```

Comment:
Voschlag zur Struktur dieser Datei (so wie es ist, finde ich schwierig zu folgen):
- Manage the workflows
   - Create a workflow
       - Define the workflow basic settings
       - Add a transition
       - Connect a transition
   - Edit a workflow
       - Select a workflow version
       - Edit a transition
       - Edit a place
   - Delete a workflow
   - Publish a workflow version
   - Unpublish a workflow verions

Reply:
Die weiteren Kapitel habe ich bewusst ausgelagert, da dass ganze viel zu unübersichtlich ist..
-> auf weitere Links verwiesen am Anfang des Kapitels